extensions/wikia/Search/WikiaSearch.setup.php

```diff
@@ -42,15 +42,14 @@
 $wgAutoloadClasses['WikiaSearchController'] =  $dir . 'WikiaSearchController.class.php';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiaSearchController" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)